web/src/components/auth/utils.js

```diff
@@ -0,0 +1,194 @@
+import axios from "axios";
+import {
+  apiUrlCreateJWTToken,
+  apiUrlRegister,
+  apiUrlLoadUser,
+  apiUrlVerifyToken,
+  apiUrlVerifyActivation,
+} from "../../api.util";
+
+/**
+ * Shape of User stored in database
+ * @typeof User
+ * @property {string} first_name
+ * @property {string} last_name
+ * @property {string} email
+ * @property {int} id
+ */
+
+/**
+ * Shape of JWT Web Token pair
+ * @typeof WebToken
+ * @property {string} refresh
+ * @property {string} access
+ */
+
+/**
+ * Stores access token in Local Storage
+ * @param {string} access the access token
+ */
+const setAccessTokenOnSuccess = (access) => {
+  localStorage.setItem("access", access);
+};
+
+/**
+ * Sets access token to null on failure in Local Storage
+ *
+ */
+const setTokensOnFailure = () => {
```

Comment:
```suggestion
const removeTokens = () => {
```

With a more generic name, we can more comfortable reuse it in other places- not just when there is a failure.